build-cmakelibs.sh

```diff
@@ -88,6 +89,7 @@ build opus
 build opusfile "-DOP_DISABLE_HTTP=ON -DOP_DISABLE_DOCS=ON -DOP_DISABLE_EXAMPLES=ON"
 build libmodplug
 build mikmod-mikmod/libmikmod "-DENABLE_SHARED=0"
+build jsoncpp  "-DBUILD_OBJECT_LIBS=OFF -DJSONCPP_WITH_TESTS=OFF -DJSONCPP_WITH_POST_BUILD_UNITTEST=OFF"
```

Comment:
you need to put it after the `cd ..` because `mikmod-mikmod/libmikmod` enters 2 sublevels.

Actually is cleaner if you do:
```sh
build mikmod-mikmod/libmikmod "-DENABLE_SHARED=0" && cd ..
build jsoncpp  "-DBUILD_OBJECT_LIBS=OFF -DJSONCPP_WITH_TESTS=OFF -DJSONCPP_WITH_POST_BUILD_UNITTEST=OFF"
```

And then remove the 
```
#we need an additional cd .. because previous library goes one sub-level more
cd ..
```

Reply:
Ah, that makes sense. I fixed the build function now to be able to deal with that.